test_databaseconnect.py

```diff
@@ -0,0 +1,151 @@
+import databaseconnect
+import config
+import pytest
+import os
+import dotenv
+import mysql.connector  as mysql
+
+dotenv.load_dotenv()
+
+class TestClass:
+    
+    @pytest.fixture(scope="session", autouse=True)
+    def setup_init(self):
+        # Will be executed before the first test
+        config.database = "test"
+        try:
+            test_db = mysql.connect(
+                host=config.host,
+                user=config.user,
+                passwd=config.password
+            )
+
+            cursor = test_db.cursor()
+            cursor.execute("CREATE DATABASE {}".format(config.database))
+            print("test database created")
+
+        except Exception:
+            print ("Failed to create test database")
+            # rolling back to orignal db
+            config.database = os.getenv("mapbotdatabase")
+            pytest.exit("Exiting pytest")
+
+        yield test_db
+
+        # Will be executed after the last test
+        try:
+            mycursor = test_db.cursor()
+            mycursor.execute("DROP DATABASE {}".format(config.database))
+            mycursor.close()
+            print("test database deleted.")
+
+        except Exception:
+            print ("Failed to delete test databse.")
+
+        config.database = os.getenv("mapbotdatabase")
+
+
+    def test_connection_to_database(self):
+        with pytest.raises(Exception) as e:
+            raise (e)
+        result = databaseconnect.connection_to_database()
```

Comment:
Again, this function simply invokes the call to connect to the db along with some exception handling. It's an attempt to test the functionality of another module and isn't quite necessary.

Reply:
okay